cmd/coordinator.go

```diff
@@ -264,6 +272,251 @@ var estimateDepositsSweepFeeCommandDescription = "Estimates the satoshi " +
 	"legacy P2SH deposits. If the estimated fee exceeds the maximum fee " +
 	"allowed by the Bridge contract, the maximum fee is returned as result"
 
+var submitDepositSweepProofCommand = cobra.Command{
+	Use:              "submit-deposit-sweep-proof",
+	Short:            "submit deposit sweep proof",
+	Long:             submitDepositSweepProofCommandDescription,
+	TraverseChildren: true,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		ctx := cmd.Context()
+
+		_, tbtcChain, _, _, _, err := ethereum.Connect(
+			ctx,
+			clientConfig.Ethereum,
+		)
+		if err != nil {
+			return fmt.Errorf(
+				"could not connect to Ethereum chain: [%v]",
+				err,
+			)
+		}
+
+		btcChain, err := electrum.Connect(ctx, clientConfig.Bitcoin.Electrum)
+		if err != nil {
+			return fmt.Errorf("could not connect to Electrum chain: %v", err)
+		}
+
+		transactionHashFlag, err := cmd.Flags().GetString(transactionHashFlagName)
+		if err != nil {
+			return fmt.Errorf("failed to find transaction hash flag: %v", err)
+		}
+
+		transactionHash, err := bitcoin.NewHashFromString(
+			transactionHashFlag,
+			bitcoin.ReversedByteOrder,
+		)
+		if err != nil {
+			return fmt.Errorf("failed to parse transaction hash flag: %v", err)
+		}
+
+		// Allow the caller to request a specific number of confirmations.
+		// The Bridge calculates the required difficulty of a chain of block
```

Comment:
This is because of this piece of code https://github.com/keep-network/tbtc-v2/blob/66a04c618e140230d25ea0d784ed75e6ba3030a2/solidity/contracts/bridge/BitcoinTx.sol#L255.

Reply:
Nice gatch and a really good comment in the code explaining it!